core/legacy/gather/driver.js

```diff
@@ -101,6 +101,20 @@ class Driver {
       rootSession: () => {
         return this.defaultSession;
       },
+      // For legacy driver, only bother supporting access to the default execution context.
```

Comment:
This seems like a reasonable time to stop trying for parity wrt legacy driver. So...just giving the default EC here.